officemanager/department/models/__init__.py

```diff
@@ -0,0 +1 @@
+from .models import *
```

Comment:
Please use absolute import instead of relative. Relative imports are not as readable as absolute ones, and it’s not easy to tell the location of the imported resources in big projects. It's a good practice to use absolute imports.